pkg/plugin/aggregation/run.go

```diff
@@ -105,7 +110,9 @@ func Run(client kubernetes.Interface, plugins []plugin.Interface, cfg plugin.Agg
 	// 5. Wait for aggr to show that all results are accounted for
 	select {
 	case <-timeout:
-		srv.Stop()
+		ctx, cancel := context.WithTimeout(context.Background(), time.Second)
```

Comment:
Why the extra second shutdown cleaning on the timeout condition vs. the hard stop on the timeout?

Reply:
because I didn't `server.Close()` the first time I read the docs 😅